tox_travis.py

```diff
@@ -67,10 +68,30 @@ def guess_python_env():
     return 'py{major}{minor}'.format(major=major, minor=minor)
 
 
+def get_default_envlist(version):
+    """Parse a default tox env based on the version.
+
+    The version comes from the ``TRAVIS_PYTHON_VERSION`` environment
+    variable. If that isn't set or is invalid, then use
+    sys.version_info to come up with a reasonable default.
+    """
+    if version in ['pypy', 'pypy3']:
+        return version
+
+    # Assume single digit major and minor versions
+    match = re.match(r'^(\d)\.(\d)(?:\.\d+)?$', version or '')
```

Comment:
this will fail in the future if a `3.10` is released, but I guess tox' `py{major}{minor}` naming system also starts to break apart then.